### DISCUSSION TITLE
Welcome to cloudproxy Discussions!

### DISCUSSION THREAD

**Discussion by claffin** [claffin/cloudproxy#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about or not sure about.
  * Share any ideas you have. 
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by L0d0c** [comment]
Hello @claffin, I wanted to know where is this project? I haven't seen any change for several months. Thanks for your help

**Reply by claffin** [reply]
Hello,

I haven't been doing many changes recently, however it still fully works. I have automated testing running in the background that tests end to end and I will fix if any issues arise.
